src/types/line.js

```diff
@@ -107,9 +107,11 @@ LineAnnotation.id = 'lineAnnotation';
 const arrowHeadsDefaults = {
   backgroundColor: undefined,
   backgroundShadowColor: undefined,
+  borderCapStyle: 'butt',
```

Comment:
Setting the default here, borderCapstyle doesn't fallback to the line options.

https://github.com/chartjs/chartjs-plugin-annotation/blob/e94475cc299fd77dd6c008b7f4bfd4a985df3a6d/src/types/line.js#L198-L208

Reply:
Thanks @stockiNail . Is this showing somewhere, like in a build output or something? I can't find a message describing why the build is failing.

Reply:
@mlohbihler my thoughts are not related to the failed build. as far as I remember, this could happen because the browser versions, used in Karma, are changed and they are managed the canvas differently comparing with the past. You need to run Karma in debug and check the differences between the rendering by browser and the fixture saved in the repo. Usually we need to increase the tolerance.